test/unit/AppFooterController.js

```diff
@@ -0,0 +1,37 @@
+describe('AppFooterController', function() {
```

Comment:
Below you have spaces before the argument braces. Could you stick to one code style?